EIPS/eip-6145.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 6145
+title: Hashtag NFT Collective Royalty Treasury
+description: A method to deposit and retrieve royalties from an Ethereum Address
+author: Ashley Turing (@livetreetech)
+discussions-to: https://ethereum-magicians.org/t/eip-6145-proposal-hashtag-nft-collective-royalty-treasury/12152
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-12-15
+---
+
+## Abstract
+
+This standard allows social media networks, NFT marketplaces and other websites to deposit royalties associated with an Ethereum address. The Ethereum address may be associated with a human readable name (e.g. a hashtag), to pay-out royalties and retrieve royalty payment information. The standard is loosely coupled and generic to maximise the potential use cases.  Unlike other EIP proposals [EIP-2981](./eip-2718.md) or [EIP-4910](./eip-4910.md) which embody mechanisms for calculating royalties, this standard sets out the mechanism for depositing and paying out royalties. The goal is to increase the earning potential for creators rather than defining the business logic for royalty payments.
```

Comment:
> Unlike other EIP proposals [EIP-2981](./eip-2718.md) or [EIP-4910](./eip-4910.md) which embody mechanisms for calculating royalties, this standard sets out the mechanism for depositing and paying out royalties.

This bit should be in motivation, not abstract. The abstract should just be a terse technical summary.